web_advanced_search_wildcard/static/src/js/search.js

```diff
@@ -0,0 +1,12 @@
+odoo.define('web_advanced_search_wildcard', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var search_filters = require('web.search_filters');
+
+    var _lt = core._lt;
+
+   search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
+            {value: '=ilike', text: _lt("matches")}
+        );
```

Comment:
Use consistent indentation please.

Reply:
do you mean as follows?

```
search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
    {value: '=ilike', text: _lt("matches")}
);
```
4 spaces are ok for indentation?

Reply:
yes exactly 😃